src/common/components/hosts/AITable.tsx

```diff
@@ -259,9 +259,13 @@ const AITable = <R extends any>({
             // parent will be set after sorting
             fullWidth: true,
```

Comment:
This needs to go away

Reply:
If we remove fullWidth:true, then the storages table looks like this:
![image](https://user-images.githubusercontent.com/11390125/201333972-236be629-0e6c-4f18-a3cb-b61c32a8b773.png)

Reply:
After talk with @jtomasek , we find that "fullWidth" is a helper which should be used only when there is a single cell in the expand. In our case we should correctly add the columns to the expand row and calculate the colspan.
I'll take a look on Monday

Reply:
@jtomasek I think I found how to do it